src/main/java/namor/template/stack/ArrayStack.java

```diff
@@ -0,0 +1,72 @@
+package namor.template.stack;
+
+import java.util.Arrays;
+
+public class ArrayStack {
+
+    private int top;
+    private int size;
+    private int[] stack;
+    private static final int SIZE = 10;
+
+    public ArrayStack(int size) {
+        if (size < 1) {
+            throw new IllegalArgumentException("Can't create stack with size less than 1");
+        }
+
+        this.size = size;
+        stack = new int[size];
+    }
+
+    public ArrayStack() {
+        this.size = SIZE;
+        stack = new int[size];
+    }
+
+    public boolean isEmpty() {
+        return top == 0;
+    }
+
+    public void push(int value) {
+        if (top == stack.length) {
+            extendStack();
+        }
+
+        stack[top] = value;
+        top++;
+    }
+
+    public int stackSize() {
```

Comment:
The class called as Stack why do you need to point this word in method name ? It's obvious that this is a size of stack.